dv/uvm/core_ibex/riscv_dv_extension/riscv_core_setting.nopmp.sv.patch

```diff
@@ -13,3 +13,32 @@
  
  // Debug mode support
  bit support_debug_mode = 1;
+@@ -144,27 +144,6 @@
+     MHPMCOUNTER8H,    // Machine performance monitoring counter (upper 32 bits)
+     MHPMCOUNTER9H,    // Machine performance monitoring counter (upper 32 bits)
+     MHPMCOUNTER10H,   // Machine performance monitoring counter (upper 32 bits)
+-    MSECCFG,          // Machine security configuration register
+-    PMPCFG0,          // PMP configuration register
+-    PMPCFG1,          // PMP configuration register
+-    PMPCFG2,          // PMP configuration register
+-    PMPCFG3,          // PMP configuration register
+-    PMPADDR0,         // PMP address register
+-    PMPADDR1,         // PMP address register
+-    PMPADDR2,         // PMP address register
+-    PMPADDR3,         // PMP address register
+-    PMPADDR4,         // PMP address register
+-    PMPADDR5,         // PMP address register
+-    PMPADDR6,         // PMP address register
+-    PMPADDR7,         // PMP address register
+-    PMPADDR8,         // PMP address register
+-    PMPADDR9,         // PMP address register
+-    PMPADDR10,        // PMP address register
+-    PMPADDR11,        // PMP address register
+-    PMPADDR12,        // PMP address register
+-    PMPADDR13,        // PMP address register
+-    PMPADDR14,        // PMP address register
+-    PMPADDR15,        // PMP address register
```

Comment:
Remove these from the `implemented_csr` list for Ibex configs that don't have PMP.